frontend/pages/RegistrationPage/RegistrationPage.tsx

```diff
@@ -50,8 +50,10 @@ const RegistrationPage = ({ router }: IRegistrationPageProps) => {
       setCurrentUser(user);
       setAvailableTeams(available_teams);
       return router.push(MANAGE_HOSTS);
-    } catch (response) {
-      console.error(response);
+    } catch (error) {
+      // TODO: Alert user to server errors
```

Comment:
@lukeheath, I discovered that the current setup UX is buggy. The final "Success" screen is misleading because the configuration has not been successfully posted to the server. In the case of a server error, the user would be stuck on the page. I retitled the page as "Confirm configuration" to be more accurate and redirect the user to the first screen in the event of a server error, which was easy enough for me to implement while I was working in the code. Because the setup forms are still using old .jsx class-based components, I didn't want to get too bogged down as far as surfacing details of the error and mucking through the old code. I figured we could spec that later if we want.